test/videojs-hls_test.js

```diff
@@ -2651,4 +2651,19 @@ test('does not download segments if preload option set to none', function() {
   strictEqual(loadedSegments, 0, 'did not download any segments');
 });
 
+test('pts value undefined test', function(){
```

Comment:
I'll rename the function. Should I be very descriptive or just an in short description will do ?